schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
+description: Represents a MW Page Delete event
+type: object
+properties:
+  # global event fields
+  meta:
```

Comment:
In [T116786](https://phabricator.wikimedia.org/T116786) the question of including the `topic` came up, enabling mixed-topic batch submissions. This would make this element even more of an `envelope`.